src/ensembl/src/content/app/browser/zmenu/ZmenuInstantDownload.tsx

```diff
@@ -15,74 +15,82 @@
  */
 
 import React from 'react';
+import { gql, useQuery } from '@apollo/client';
+import { Pick2 } from 'ts-multipick';
 
-import useApiService from 'src/shared/hooks/useApiService';
+import { parseFeatureId } from '../browserHelper';
 
 import { InstantDownloadTranscript } from 'src/shared/components/instant-download';
 import { CircleLoader } from 'src/shared/components/loader/Loader';
-import { TranscriptInResponse } from 'src/content/app/entity-viewer/shared/rest/rest-data-fetchers/transcriptData';
 
-import { LoadingState } from 'src/shared/types/loading-state';
+import { FullTranscript } from 'ensemblRoot/src/shared/types/thoas/transcript';
```

Comment:
🙅‍♂️ `ensemblRoot` (unfortunately, VSCode inserts it automatically if you rely on it for module imports)

Reply:
There are lots of ensemblRoot I could see in our code. Should we remove all of them?

Reply:
Some of those `ensemblRoots` are useful, which is why we created the alias. Let's just clean this PR from them.